api/utils.py

```diff
@@ -87,14 +87,16 @@ def proxy_to(to_url):
 
 
 def build_flask_app(name):
-    if running_local():
-        app = Flask(name)
-        CORS(app)
-    else:
-        app = Flask(name, static_url_path="", static_folder="frontend/public")
+    is_running_local = running_local()
+
+    additional_args = {}
+    if is_running_local:
+        additional_args = dict(static_url_path="", static_folder="frontend/public")
```

Comment:
issue(blocking): careful, the additional args belong to the "non-local" branch, if I understand correctly

Reply:
Thank you, fixed in the next rev!